spec/message.abnf

```diff
@@ -49,13 +49,16 @@ reserved-char  = %x00-08        ; omit HTAB and LF
 
 name    = name-start *name-char ; based on https://www.w3.org/TR/xml/#NT-Name, but cannot start with U+003A COLON ":"
 nmtoken = 1*name-char           ; equal to https://www.w3.org/TR/xml/#NT-Nmtoken
+argtoken = arg-start *name-char
 name-start = ALPHA / "_"
            / %xC0-D6 / %xD8-F6 / %xF8-2FF
            / %x370-37D / %x37F-1FFF / %x200C-200D
            / %x2070-218F / %x2C00-2FEF / %x3001-D7FF
            / %xF900-FDCF / %xFDF0-FFFD / %x10000-EFFFF
 name-char = name-start / DIGIT / "-" / "." / ":"
           / %xB7 / %x0300-036F / %x203F-2040
+arg-start = name-start / DIGIT / "."
+          / %xB7 / %x0300-036F / %x203F-2040
```

Comment:
technically `%x0300-036F` in both of these productions should be `%x300-36F`. The Unicadet in me prefers four digits (except when five or six are needed), but we should be consistent in our ABNF and with common ABNF practices.

Reply:
Done.